### PR TITLE
Arabic trans

### PR DESCRIPTION
**Pull request by EmanHerawy**
## Description
Ethereum.org Site Translation: Arabic #541 following all the steps mentioned in this bounty https://ethscholars.bounties.network/bounty/3747

## Related Issue
Solves Issue #541

### CODE REVIEW

**Comment by samajammin** [comment]
@all-contributors please add @EmanHerawy for code

File: docs/.vuepress/theme/components/Hero.vue [hunk 1]
```diff
@@ -58,6 +58,7 @@ export default {
         'Αιθέριο',
         'Eterijum',
         'إثيريوم',
+        'إثيريوم',
```
[the remark this answers is not in the document]
Comment (EmanHerawy) [reply docs/.vuepress/theme/components/Hero.vue]: Sorry about that , I didn't expect it was already added. I have fixed it . I have to create new PR, Right ?
```suggestion
        'إثيريوم',
```

**Comment by samajammin** [comment]
Hey @EmanHerawy - please head here to fulfill the work: https://ethscholars.bounties.network/bounty/3747

Once that's complete, we'll get this bounty paid out to you! Thanks again.

**Comment by EmanHerawy** [comment]
@samajammin  Thanks a lot. I have already fulfilled the bounty

**Pull request merged**